openedx/features/assessment/tests/test_helpers.py

```diff
@@ -0,0 +1,247 @@
+from datetime import datetime
+
+import mock
+from django.contrib.auth.models import User
+from pytz import UTC
+from submissions.models import Submission
+
+from openedx.features.assessment import helpers
+from openedx.features.assessment.tests.factories import AnonymousUserIdFactory, StudentItemFactory, SubmissionFactory
+from student.tests.factories import CourseEnrollmentFactory
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+
+from .constants import (
+    FOUR_DAYS_AGO,
+    PHILU_BOT_NAME,
+    TEST_RUBRIC_CRITERIA,
+    THREE_POINT_RUBRIC_DICTIONARY,
+    THREE_WEEKS_AGO,
+    TWO_POINT_RUBRIC_DICTIONARY
+)
+from .helpers import create_source_course
+
+
+class AssessmentHelperTestCases(ModuleStoreTestCase):
+
+    def _create_user_enrollments(self, user, created_date=None):
+        """ Create course enrollment """
+        enrollment = CourseEnrollmentFactory(
+            course_id=self.source_course.id,
+            user=user,
+        )
+        if created_date:
+            enrollment.created = created_date
+            enrollment.save()
+
+        return enrollment
+
+    def setUp(self):
+        super(AssessmentHelperTestCases, self).setUp()
+
+        self.source_course = create_source_course(self.store, datetime(2019, 9, 1, tzinfo=UTC))
+
+        self.enrollment = self._create_user_enrollments(self.user, THREE_WEEKS_AGO)
+
+        self.course_chapters = modulestore().get_items(
+            self.source_course.id,
+            qualifiers={'category': 'course'}
+        )
+
+    @mock.patch('openedx.features.assessment.helpers.log.info')
+    def test_log_multiple_submissions_info_with_multiple_submissions(self, mock_log_info):
+        SubmissionFactory.create()
+        SubmissionFactory.create()
+        submissions = Submission.objects.all()
+        helpers._log_multiple_submissions_info(submissions)
+        assert mock_log_info.called_once
```

Comment:
It is better if we can check `mock_log_info` contains a substring in log string i.e "ids 1,3,4"

Reply:
done